.ado/publish.yml

```diff
@@ -107,7 +107,7 @@ jobs:
   - job: RNMacOSInitNpmJSPublish
     displayName: react-native-macos-init Publish to npmjs.org
     pool:
-      vmImage: vs2017-win2016
+      vmImage: ubuntu-18.04
```

Comment:
sorry I don't know much about macos' pipelines, but is it ok to swap the win one for Ubuntu?

Reply:
I think so. I think us using windows here is a relic of this repo's pre open-source days because nothing in this pipeline seems windows specific. In any case, It's hard to test publish pipelines effectively, so I'd have to keep a close eye in case it _does_ break.